src/style.css

```
@@ -0,0 +1,73 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+.box {
+  margin-top: 20px;
+}
+
+.container {
+  display: block;
+}
+
+.input {
+  display: inline-block;
+  font-family: Avenir, sans-serif;
+  padding: 0;
+  outline: none;
+  width: 100%;
+  color: #3d4e61;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.big_input {
+  font-weight: 300;
+  padding-left: 62px;
+  height: 70px;
+  font-size: 16px;
+  line-height: 22px;
+}
+
+.small_input {
```

```suggestion
.input--small {
```
2 point in checklist

---

could you please write why it is better to use "--" than "_" in the class name

---

because this is a modifier, you will learn more about it when you will start using BEM